pages.ko/windows/mkdir.md

```diff
@@ -0,0 +1,12 @@
+# mkdir
+
+> 이 명령은 디렉터리를 생성합니다.
+> 자세한 내용 : <https://learn.microsoft.com/windows-server/administration/windows-commands/mkdir>
+
+- 디렉토리 만들기:
+
+`mkdir {{디렉토리 이름}}`
+
+- 중첩된 디렉토리 트리를 재귀적으로 만듭니다:
+
+`mkdir {{서브디렉토리 이름 경로}}`
```

Comment:
```suggestion
`mkdir {{경로/대상/서브디렉토리}}`
```
Referred to the common arguments file and Google translate. Feel free to reword it.